docs/pre-release/reference/intercepts.md

```diff
@@ -40,6 +40,17 @@ This will intercept a workload named `hello` and name the intercept `myhello`.
 
 Telepresence can import the environment variables from the pod that is being intercepted, see [this doc](../environment/) for more details.
 
+## Creating an intercept without a local process running
+
+If you are *are* logged into Ambassador Cloud, when you create an intercept the Traffic
```

Comment:
are *are* ?

Reply:
I would suggest a more active wording, starting with "The Traffic Manager sends a GET request ..."